GUIDELINES.md

```diff
@@ -0,0 +1,70 @@
+Spider writing guidelines
+=========================
+
+Before start working on a spider, read carefully the following guidelines that will make the project more consistent and your PR more probable to be accepted.
+
+Reading code of existing spiders will also help you to understand how to develop.
+
+Spider class
+------------
+
+- All spiders must inherit from `BaseGazetteSpider`
```

Comment:
This is not entirely true, `FecamGazetteSpider` is a base class that can be used and someone could also create a new base spider, it may be relevant to note that.

Of course, all spiders and base spiders have to inherit from `BaseGazetteSpider` at some point.

Reply:
This is true, but I believe this is a more advanced use of base classes, so only more advanced contributors will worry about that.